corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -360,6 +361,8 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             } else {
                 self.selectedCaseIds = [];
             }
+
+            self.showMap = !!_.find(this.styles, function (style) { return style.displayFormat === "Address"; });
```

Comment:
How about making "Address" a constant?

Reply:
Is there a standard rule for when something should be formatted as a constant?

Reply:
There isn't a team guideline. My own opinion is that any value that isn't 0 or 1 should probably be a constant, for the sake of readability and having your IDE catch your spelling errors. But I might be an ideologue.